src/main/java/roomescape/domain/Reservation.java

```diff
@@ -28,23 +30,30 @@ public class Reservation {
     @ManyToOne(fetch = FetchType.LAZY)
     private Theme theme;
 
-    public Reservation(Long id, Member member, LocalDate date, ReservationTime reservationTime, Theme theme) {
-        validate(member, date, reservationTime, theme);
+    @Enumerated(EnumType.STRING)
+    private ReservationStatus reservationStatus;
+
+    public Reservation(Long id, Member member, LocalDate date, ReservationTime reservationTime,
+                       Theme theme, ReservationStatus reservationStatus) {
```

Comment:
```suggestion
    public Reservation(
        Long id, 
        Member member,
        LocalDate date, 
        ReservationTime reservationTime,
        Theme theme, 
        ReservationStatus reservationStatus
    ) {
```
지금 방법은 어떤 파라미터가 있는지 파악하기 힘든것 같아요ㅠ
파라미터가 여러개라서 120자를 넘어간다면 각 파라미터마다 개행을 하는걸 추천드립니다

Reply:
이 부분도 반영했습니다!